plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryTypeMapping.java

```diff
@@ -526,44 +533,122 @@ public void testTime()
                 .execute(getQueryRunner(), bigqueryViewCreateAndInsert("test.time"));
     }
 
-    @Test
-    public void testTimestampWithTimeZone()
+    @Test(dataProvider = "zonesDataProvider")
+    public void testTimestampWithTimeZone(ZoneId zoneId)
     {
+        Session session = Session.builder(getSession())
+                .setTimeZoneKey(TimeZoneKey.getTimeZoneKey(zoneId.getId()))
+                .build();
+
         SqlDataTypeTest.create()
                 // min value in BigQuery
-                .addRoundTrip("TIMESTAMP", "TIMESTAMP '0001-01-01 00:00:00.000000 UTC'",
+                .addRoundTrip("TIMESTAMP", "TIMESTAMP '0001-01-01 00:00:00.000'",
```

Comment:
These tests are broken as far as I run locally.

Reply:
Test is passing now